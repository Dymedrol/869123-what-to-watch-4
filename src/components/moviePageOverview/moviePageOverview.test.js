import React from 'react';
import renderer from 'react-test-renderer';

import MoviePageOverview from './moviePageOverView.jsx';
import MOVIES from "../../mocks/films.js";

const movie = MOVIES[1];

it(`Проверяет снепшот компонента FilmPageDetalis`, () => {
  const tree = renderer.create(
      <MoviePageOverview
        movie={movie}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
