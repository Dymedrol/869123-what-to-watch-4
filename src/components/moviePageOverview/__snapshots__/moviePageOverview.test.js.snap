// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Проверяет снепшот компонента FilmPageDetalis 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      7.9
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        good
      </span>
      <span
        className="movie-rating__count"
      >
        94
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p>
      A pair of young lovers flee their New England town, which causes a local search party to fan out to find them.
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Wes Anderson
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Jared Gilman, Kara Hayward, Bruce Willis
         and other
      </strong>
    </p>
  </div>,
]
`;
